docs/source/release_notes/release-notes.rst

```diff
@@ -34,39 +34,47 @@ Changelog
 develop
 =======
 
+.. <<<<------------------------------------------------------------------------------------------------------------->>>>
+
+=======
+v0.26.0
+=======
+
 .. list-table::
     :widths: 5 40
     :header-rows: 1
 
     *    - Type
          - Change
 
-    *    - |breaking|
```

Comment:
I would have this in, but as `|fixed|`. Might help us recall that this part of our HA story was fixed in this particular version.